spec/request/user_post_request_spec.rb

```diff
@@ -0,0 +1,17 @@
+# spec/request/user_post_request_spec.rb
+
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users/1/posts'
+
+      expect(response.status).to eq(200)
+
+      expect(response).to render_template('index')
+
+      expect(response.body).to include('This is posts page index')
+    end
+  end
+end
```

Comment:
You did a great job working on this milestone! 💯 However, here are some general suggestions:

- [ ] Instead of having separate files for each action e.g. `index`, `show`. I suggest you have only one, let's say `posts_spec.rb` and have two separate describe blocks, like this:

```rb
 RSpec.describe "Posts", type: :request do
  describe 'GET /index' do
    ...
  end

  describe 'GET /show' do
    ...
  end
 end

```

- [ ] **[OPTIONAL]** Also, It's a good practice to have one assert (expect) per test case, so it would be good if you consider refactoring your test suite to meet this suggestion, something like this:

```rb
RSpec.describe 'Posts', type: :request do
  describe 'GET /index' do
    before { get '/users/1/posts' }

    it 'returns http success' do
      expect(response.status).to eq(200)
    end

    it 'renders the correct template' do
      expect(response).to render_template('index')
    end

    it 'renders the correct placeholder text' do
      expect(response.body).to include('This is posts page index')
    end
  end
...
end
```


Please consider applying these suggestions to both `users` and `posts`! 🤓 🔧